Bojzen-I-Mitten/Stort-spelprojekt
# Toy soldier powerup

* Added new powerup (toy soldier)

*** HOW TO TEST ***
* Get the latest SVN
* Build out the game and open two client
* Pick up the toy soldier powerup if you get it as spawn and then move closer
to the other chad and throw it for the powerup effect
* You can try running around as the powerup-affected chad and notice that
the movement is slowed down and that you can't pick up objects
* After 10 seconds the effect will decay and the affected chad will be back to normal state

*** KNOWN ISSUES ***
* When the collider is reset for the previously affected chad there may be some weird
graphical resize issues if you resize at certain places
* There is no despawn time for the powerup (Nick will fix this)
* The other chad can catch the toy soldier if you throw it at his hand (this may be the intention?)
